search/api.py

```diff
@@ -368,11 +368,18 @@ def transform_results(search_result, user):
         "audience",
         "certification",
         "department_name",
+        "level",
     ]:
         if f"agg_filter_{aggregation_key}" in search_result.get("aggregations", {}):
-            search_result["aggregations"][aggregation_key] = search_result[
-                "aggregations"
-            ][f"agg_filter_{aggregation_key}"][aggregation_key]
+            if aggregation_key == "level":
```

Comment:
This is not working as it's supposed to.

For example, if you filter for topic "Game Design" with the courses pulled by `npm run import:ocw:exampe` from the rc bucket you still get 28 undergraduate classes and 2 graduate classes in the level facet instead of 15 and 2.

The data structure returned under aggregations is 
```
'agg_filter_level': {'doc_count': 16, 'level': {'doc_count': 30, 'level': {'doc_count_error_upper_bound': 0, 'sum_other_doc_count': 0, 'buckets': [{'key': 'Undergraduate', 'doc_count': 28, 'courses': {'doc_count': 15}}, {'key': 'Graduate', 'doc_count': 2, 'courses': {'doc_count': 2}}]}}}
```